views/common.py

```diff
@@ -34,7 +34,7 @@ def display_errors(request, errors):
             messages.add_message(request, messages.ERROR, "{} : {}".format(_(key), value[0]), "alert-danger")
 
 
-def display_report_errors(request,errors):
+def display_report_errors(request, errors):
```

Comment:
CC : Function `display_report_errors` has a Cognitive Complexity of 7 (exceeds 5 allowed).